src/chi/components/card/card.scss

```diff
@@ -27,6 +26,10 @@
       .a-card__title {
         margin: 0 0 0.5rem 0;
       }
+
+      & p {
```

Comment:
@davidojedacl why attach the element to p. What happen if any developer insert another html tag?

Reply:
if we remove it can happen this. 
<img width="628" alt="Screen Shot 2019-03-13 at 4 09 13 PM" src="https://user-images.githubusercontent.com/43439742/54289955-68695080-45aa-11e9-89b6-0e0f260a011d.png">

So also can attach to a-card__title also or leave it to all except when the element would be Input -toggle.

@mattnickles Thoughts?